assets/javascripts/discourse/components/chat-composer.js

```diff
@@ -680,6 +680,24 @@ export default Component.extend(TextareaTextManipulation, {
     this.onValueChange?.(this.value, this._uploads, this.replyToMsg);
   },
 
+  @action
+  onTextareaFocusin(target) {
```

Comment:
Very minor but I would uppercase the I in "In" since it's two words "Focus In"

```suggestion
  onTextareaFocusIn(target) {
```